### PR TITLE
docs(examples): static CodeSandbox examples

### PR DESCRIPTION
**Pull request by asudoh**
### Description

Moves our example of using pre-built Web Components bundle to CodeSandbox.

Also introduces a CodeSandbox example to use the pre-built bundle with Handlebars as an example of server-oriented application. Given we have more than one Handlebars examples, created a dedicated document page for those Handlebars examples. This CodeSandbox revealed that `@carbon/ibmdotcom-services` package depends on `@babel/runtime`. Added such dependency.

### Changelog

**New**

- A CodeSandbox example to use the pre-built bundle with Handlebars.
- Explicit `@babel/helpers` dependency in `@carbon/ibmdotcom-services`.

**Changed**

- Moves our example of using pre-built Web Components bundle to CodeSandbox.

### CODE REVIEW

File: packages/web-components/README.md [hunk 1]
```diff
@@ -49,8 +49,7 @@ For quick start, you can use our pre-built bundle that contains masthead, footer
 <html>
   <head>
     <script type="module">
-      // Copy from `dist` directory in the package and put it to the same directory as this file
-      import './ibmdotcom-web-components-dotcom-shell.min.js';
+      import 'https://www.ibm.com/common/carbon-for-ibm-dotcom/v0.5.0/ibmdotcom-web-components-dotcom-shell.min.js';
```
Comment (jeffchew) [inline packages/web-components/README.md:52]: @asudoh you can use:
`https://www.ibm.com/common/carbon-for-ibm-dotcom/latest/ibmdotcom-web-components-dotcom-shell.min.js`

File: packages/web-components/docs/server-side-template.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/web-components/examples/codesandbox/usage/handlebars)
+
+While we recommend building front-end JavaScript/Sass code with a module bundler along with server-side technology like Handlebars (it's what above example does), you can go with completely server-side technology for your quick start, by using our pre-built bundle, like:
+
+```handlebars
+<html>
+  <head>
+    <script type="module">
+      // Uses pre-built bundle
+      import 'https://www.ibm.com/common/carbon-for-ibm-dotcom/v0.5.0/ibmdotcom-web-components-dotcom-shell.min.js';
```
Comment (jeffchew) [inline packages/web-components/docs/server-side-template.md:75]: Same for here:
`https://www.ibm.com/common/carbon-for-ibm-dotcom/latest/ibmdotcom-web-components-dotcom-shell.min.js`

File: packages/web-components/examples/codesandbox/usage/bundle-handlebars/index.hbs [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+      main {
+        margin: calc(3rem + 1px) auto 0 auto;
+        padding: 3rem;
+        max-width: 99rem;
+      }
+    </style>
+  </head>
+  <body>
+    <dds-masthead>
```
Comment (jeffchew) [inline packages/web-components/examples/codesandbox/usage/bundle-handlebars/index.hbs:86]: @asudoh just wondering if an easier example could be used like the `dds-dotcom-shell-container` instead?

Comment (asudoh) [reply packages/web-components/examples/codesandbox/usage/bundle-handlebars/index.hbs:86]: @jeffchew Good question, this example is of a server-side rendering whereas `<dds-masthead-container>` does client-side rendering.

File: packages/web-components/README.md (hunk 1 above)
Comment (asudoh) [reply packages/web-components/README.md]: Thanks @jeffchew! Updated.

File: packages/web-components/docs/server-side-template.md (hunk 2 above)
Comment (asudoh) [reply packages/web-components/docs/server-side-template.md]: Thanks @jeffchew! Updated.

**Review by jeffchew — APPROVED** [review]
LGTM!

**Pull request merged**